--- a/usr/share/lipstick-jolla-home-qt5/notifications/NotificationColumn.qml
+++ b/usr/share/lipstick-jolla-home-qt5/notifications/NotificationColumn.qml
@@ -42,10 +42,6 @@
                     "property": "category",
                     "comparator": "!match",
                     "value": "^x-nemo.system-update"
-                }, {
-                    "property": "priority",
-                    "comparator": ">=",
-                    "value": 100
                 } ]
                 if (Screen.sizeCategory >= Screen.Large) {
                     // Temporary: don't show missed calls on tablet
